libafl/src/feedbacks/list.rs

```diff
@@ -64,6 +64,47 @@ libafl_bolts::impl_serdeany!(
     <u8>,<u16>,<u32>,<u64>,<i8>,<i16>,<i32>,<i64>,<bool>,<char>,<usize>
 );
 
+impl<T> ListFeedback<T>
+where
+    T: Debug + Serialize + Hash + Eq + DeserializeOwned + Default + Copy + 'static,
```

Comment:
i think here not all of these bounds are necessary.

Reply:
Seems like they are

Reply:
(removed all of them and needed to add back one by one)